src/meta/src/rpc/service/ddl_service.rs

```diff
@@ -688,45 +689,57 @@ where
                 .get(kafka_props_broker_key(properties))
                 .cloned()
                 .ok_or(MetaError::from(anyhow!(
-                    "Must specify brokers in WITH clause",
+                    "Must specify brokers property in WITH clause",
```

Comment:
```suggestion
                    "Must specify \"properties.bootstrap.server\" in WITH clause",
```

By the way, why is there a `properties.`? Looks redundant..

Reply:
it is following Flink's naming style